packages/editor-ui/src/store.ts

```diff
@@ -708,6 +712,30 @@ export const store = new Vuex.Store({
 			const updated = state.sidebarMenuItems.concat(menuItems);
 			Vue.set(state, 'sidebarMenuItems', updated);
 		},
+		setCurrentWorkflowExecutions (state: IRootState, executions: IExecutionsSummary[]) {
+			state.currentWorkflowExecutions = executions;
+		},
+	},
+	actions: {
+		async loadCurrentWorkflowActions(context: ActionContext<IRootState, IRootState>, workflowId: string) {
```

Comment:
nitpick here but `loadCurrentWorkflowExecutions` should get workflow id from store.. But `loadWorkflowExecutions` should get workflow id as prop.. Current implies open workflow.

Reply:
👍